assets/userCtrl.js

```diff
@@ -475,4 +478,18 @@ module.exports = function ($scope, $filter, $location, UserFactory) {
   $scope.deleteFlair = function (index) {
     $scope.flairs.splice(index, 1);
   };
+
+  $scope.getFlairs();
+
+  // Get a CSRF token from the server.
+  io.socket.get('/csrfToken', function (token) {
+    $scope._csrf = token._csrf;
+  });
+
+  /* Include the csrf token in all POST requests.
+   * (As far as I can tell, there's no better way to do this aside from manually adding the token parameter to each request.)*/
+  io.socket.post = function (url, data, callback) {
+    data._csrf = $scope._csrf;
+    io.socket.request({method: 'post', url: url, params: data}, callback);
+  };
```

Comment:
Maybe add an angular service to override this rather than putting it in this controller?